tests/integration/reqs/test_account_objects.py

```diff
@@ -1,15 +1,23 @@
-from unittest import TestCase
+from unittest import IsolatedAsyncioTestCase
 
-from tests.integration.it_utils import JSON_RPC_CLIENT
+from tests.integration.it_utils import ASYNC_JSON_RPC_CLIENT, JSON_RPC_CLIENT
 from tests.integration.reusable_values import WALLET
 from xrpl.models.requests import AccountObjects
 
 
-class TestAccountObjects(TestCase):
-    def test_basic_functionality(self):
+class TestAccountObjects(IsolatedAsyncioTestCase):
+    def test_basic_functionality_sync(self):
         response = JSON_RPC_CLIENT.request(
             AccountObjects(
                 account=WALLET.classic_address,
             )
         )
         self.assertTrue(response.is_successful())
+
+    async def test_basic_functionality_async(self):
```

Comment:
i was trying to find a way to accomplish what you have here using decorators. i think it's worth it so that we don't have to duplicate code in the future. would you look into that please?